spec/models/data_request_spec.rb

```diff
@@ -222,29 +238,13 @@
   describe "#request_type" do
     context "with valid values" do
       it "does not error" do
```

Comment:
Thought it might be better to use this loop to capture all requests, the list is starting to get big